### DISCUSSION TITLE
Warehouse/Granary setting for Imperial Requests

### DISCUSSION THREAD

**Discussion by BazzaCuda** [Keriew/augustus#582] [Ideas]
For warehouses and granaries, we can set who has access to the goods they stock.
It would be good to have the ability to also designate whether the warehouse's/granary's goods can be used to fulfill imperial requests.

It's really annoying when I set up a side-industry dedicated to fulfilling such requests, only for my "Send Request" to empty one of the warehouses near to a market and leave my dedicated warehouse stocked to the brim. It would be good to have control over this, as would be the case in real life.

p.s. are imperial requests, e.g. wheat, ever fulfilled from granaries or is it always from warehouses?

**Comment by crudelios** [comment]
I like this idea.

Food is always first fulfilled from warehouses, and it only fulfills from granaries if you don't have enough food stored in the warehouse. However, when fulfilling from granaries, it will always ask you first.

**Reply by BazzaCuda** [reply]
Thanks for the clarification. My bugbear is primarily with goods like pottery, furniture, oil and weapons being taken from my city/market warehouses rather than from the dedicated fulfillment warehouses I've created, but it would be just as annoying if it were food taken from my market granaries - it's good to know there's at least some control of the granary stocks.
